src/components/command-bar/commands/search/components/tutorials-tab-contents/tutorial-hit.tsx

```diff
@@ -0,0 +1,30 @@
+import { productSlugsToNames } from 'lib/products'
+import { CommandBarLinkListItem } from 'components/command-bar/components'
+import { ProductSlug } from 'types/products'
+import { TutorialHitObject, TutorialHitProps } from './types'
+
+const TutorialHit = ({ hit }: TutorialHitProps) => {
+	const { _highlightResult, id, products, defaultContext, slug } = hit
+	const { name, description } = _highlightResult
+	const badges = products?.map(
+		(productSlug: ProductSlug) => productSlugsToNames[productSlug]
+	)
+	const [productSlug, collectionSlug] = defaultContext.slug.split('/')
+	const [, tutorialSlug] = slug.split('/')
+	const resultUrl = `/${
+		productSlug === 'cloud' ? 'hcp' : productSlug
+	}/tutorials/${collectionSlug}/${tutorialSlug}`
+
+	return (
+		<CommandBarLinkListItem
+			key={id}
```

Comment:
suggestion: As these aren't being rendered as part of a map, this `key` prop doesn't have any practical usage. I see you are providing a key when rendering `<TutorialHit>`, so we're covered there!

Reply:
Oop! Those a copy-pasta remnants! Removing 😎

Reply:
Resolved in: https://github.com/hashicorp/dev-portal/pull/990/commits/8c8f40caded817e6bda10e2b52beb1c94dac3085